src/integration/pages/mostWatchedPage/ampTests.js

```diff
@@ -0,0 +1,8 @@
+import { runCoreAmpTests, runAmpAnalyticsTests } from '../../common';
+import runCrossPlatformTests from './crossPlatformTests';
+
+export default service => {
```

Comment:
`service` is a global in all int test files

Reply:
Aye it seems like we have been passing it down in other integration tests, but there's no reason to.

@pharingee - without the passing of service as props works fine https://github.com/bbc/simorgh/commit/15ce277aba750892470fc0c2a150cebe46a9ca13